tests/plan_instances.py

```diff
@@ -0,0 +1,94 @@
+from djstripe.models import Plan
```

Comment:
I import plans from "plan_instances.py" which are used where it is necessary not to duplicate code. maybe you have a more flexible solution?